user_guide/en.html

```diff
@@ -131,7 +133,7 @@
     background-color: var(--container-color);
     padding-left: 3%;
     padding-right: 3%;
-    padding-top: 1%;
+    padding-top: 0%;
```

Comment:
As an aside, any 0 doesn't need a type of value for obvious reasons. I typically prefer to leave them out.

Reply:
You are right Frenzie. I forgot that this container is top level and that property doesn't need overriding. Can you delete that line, I don't know how to do here.

Reply:
Oh, I actually just meant this:
```suggestion
    padding-top: 0;
```

But we can just remove the line completely here if it's not necessary:
```suggestion
```

You can make a suggestion with the + icon next to write/preview, after which you can easily apply it. For small things it can be the most convenient.